odm2api/ODM2/services/readService.py

```diff
@@ -933,6 +933,8 @@ def getDataSetsValues(self, ids=None, codes=None, uuids=None, dstype=None):
             uuids (list, optional): List of Dataset UUIDs string.
             dstype (str, optional): Type of Dataset from
                 `controlled vocabulary name <http://vocabulary.odm2.org/datasettype/>`_.
+            lowercols (bool, optional): Make column names to be lowercase.
+                                        Default to True.
```

Comment:
Add this at the end of the `lowercols` description: "Please start upgrading your code to rely on CamelCase column names. In a near-future release, the default will be changed to False, and later the parameter may be removed.